x-pack/plugins/apm/public/components/shared/date_picker/index.tsx

```diff
@@ -25,7 +25,7 @@ export function DatePicker({
   rangeTo?: string;
   refreshPaused?: boolean;
   refreshInterval?: number;
-  onTimeRangeRefresh: (range: { start: string; end: string }) => void;
```

Comment:
why did this change?

Reply:
I realised that these parameters are not used at all. So removed them.